Integrations/GoogleCloudStorage/GoogleCloudStorage.py

```diff
@@ -0,0 +1,566 @@
+import traceback
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+from google.cloud import storage
+
+import datetime
+import json
+import re
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+''' GLOBALS/PARAMS '''
+
+RFC3339_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DEMISTO_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+SERVICE_ACCOUNT_JSON = demisto.params()["service_account_json"]
+USE_PROXY = demisto.params().get("use_proxy")
+
+
+def safe_del(dictionary, key):
+    if key in dictionary:
+        del dictionary[key]
+
+
+# Remove proxy if not set to true in params
+if not USE_PROXY:
+    safe_del(os.environ, "HTTP_PROXY")
+    safe_del(os.environ, "HTTPS_PROXY")
+    safe_del(os.environ, "http_proxy")
+    safe_del(os.environ, "https_proxy")
+
+
+''' HELPER FUNCTIONS '''
+
+
+def init_storage_client():
+    cur_directory_path = os.getcwd()
+    credentials_file_name = "{}.json".format(demisto.uniqueFile())
```

Comment:
use `f''` instead of `.format()`

Reply:
Is this necessary? I'm aware of `f''` but I find this style nicer.